printer.template.cfg

```diff
@@ -24,23 +24,24 @@ serial: /dev/serial0 # If you are using internal RPI serial port
 restart_method: command
 
 ### CONTROL BOARD
-[include klipper-prusa-mk3s/mk3s/einsy-rambo.cfg]
+[include ./mk3s/einsy-rambo.cfg]
 
 ### BASE SETUP
-[include klipper-prusa-mk3s/mk3s/display.cfg]
-[include klipper-prusa-mk3s/mk3s/steppers.cfg]
-[include klipper-prusa-mk3s/mk3s/tmc2130.cfg]
+[include ./mk3s/display.cfg]
+[include ./mk3s/steppers.cfg]
+[include ./mk3s/tmc2130.cfg]
 
 ### EXTRUSION
 
 # Extruder
-[include klipper-prusa-mk3s/extruders/prusa.cfg]
-# [include klipper-prusa-mk3s/extruders/bmg.cfg]
+[include ./extruders/prusa.cfg]
+# [include./extruders/bmg.cfg]
 
 # Hotend
-[include klipper-prusa-mk3s/hotends/v6.cfg]
-# [include klipper-prusa-mk3s/hotends/dragon-standard-flow.cfg]
-# [include klipper-prusa-mk3s/hotends/rapido.cfg]
+[include ./hotends/v6.cfg]
+# [include ./hotends/dragon-standard-flow.cfg]
+# [include ./hotends/dragon-high-flow.cfg]
+# [include ./hotends/rapido.cfg]
 
```

Comment:
These include paths will probably not work with the current recommended installation instructions.